source/docs/software/actuators/solenoids.rst

```diff
@@ -0,0 +1,102 @@
+Operating pneumatic cylinders
+=============================
+
+Solenoid control
+----------------
+
+FRC teams use solenoids to preform a variety of tasks, from shifting
+gearboxes to operating robot mechanisms. A solenoid is a valve used to
+electronically switch a pressurised air line "on" or "off". For more
```

Comment:
Not completely true.  We do have and allow electric solenoid actuators.  This sentence should clarify this distinction.

Reply:
Adding in a short blurb about this being focused on pneumatics, but the electrical solenoids are legal.